### ISSUE TITLE
Segmentation fault!

### ISSUE THREAD

**Issue by muxiddin19** [srush/annotated-s4#80]
Hi! Great job!
Can you help me with this issue?
(ann1_s4) wsl@DESKTOP-CJL0ECN:/mnt/h/muhiddin/annotated-s4$ python -m s4.train dataset=quickdraw layer=s4 train.epochs=10 train.bsz=128 model.d_model=128 model.layer.N=64
2024-05-21 13:17:39.628139: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
dataset: quickdraw
layer: s4
seed: 0
model:
  d_model: 128
  n_layers: 4
  dropout: 0.0
  prenorm: true
  embedding: false
  layer:
    'N': 64
train:
  epochs: 10
  bsz: 128
  lr: 0.001
  lr_schedule: false
  weight_decay: 0.01
  checkpoint: false
  suffix: null
  sample: null
wandb:
  mode: disabled
  project: s4
  entity: null

[*] Warning: models are not being checkpoint
[*] Setting Randomness...
Segmentation fault

Whatever training code I run gives me this issue. I would appreciate any help!
